lib/multer-sharp/multer-sharp.ts

```diff
@@ -15,6 +15,7 @@ import {
   transformImage,
   isOriginalSuffix,
 } from './multer-sharp.utils';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
How about importing `randomStringGenerator` method from the Nest common utils instead?

```suggestion
import { randomStringGenerator } from '@nestjs/common/utils/random-string-generator.util';
```

Reply:
How about using template literals? They allow for both multiline strings and string interpolation.
```typescript
file.originalname =  `${randomStringGenerator()}.${extension(mimetype)}`
```